app/lib/google_scraper_ruby/errors.rb

```diff
@@ -0,0 +1,46 @@
+# frozen_string_literal: true
+
+module GoogleScraperRuby
+  module Errors
+    class SearchServiceError < StandardError
+      def initialize(url:, error: nil)
+        @original_error = error
+        @url = url
+        super(error)
+      end
+
+      def to_s
+        <<~ERR_MESSAGE
+          Failed to perform search request with URL: #{url}
+          - original_error: #{original_error}"
+        ERR_MESSAGE
+      end
+
+      private
+
+      attr_reader :original_error, :url
+    end
+
+    class ScrapeServiceError < StandardError
```

Comment:
in Ruby we would prefer to have separate file for each class instead of having a single file for all classes @Goose97 so let's move this to 2 files 

- app/lib/google_scraper_ruby/errors/search_service_error.rb
- app/lib/google_scraper_ruby/errors/scrape_service_error.rb

I also recommend to remove the `_service_` in the file name, so it's just about the `search_error` or `scrape_error` what do you think?

Reply:
Fixed in https://github.com/Goose97/google-scraper-ruby/pull/20/commits/54cff3f50e72c4bd4d34390f93d1c25caf25b5bc

Reply:
@Goose97 the fix is just about rename the class, both class still in the same file, please double check my above comment 🙏

Reply:
my bad. Fixed in 8d8f1ce6d19a1481c0ac1e8b21c7c935ce4f4bd0